{"uuid": "a140129c-163c-4e0d-83ed-b8c28d4aa310", "children": ["d10bf997-732d-4fe9-a801-f60b9ab55ce5", "74166278-e037-482b-b20d-66a4e03a65ad"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_INTERNET_DISCONNECTED\n  (Session info: chrome=130.0.6723.92)\nStacktrace:\n\tGetHandleVerifier [0x00007FF7E5D83AF5+28005]\n\t(No symbol) [0x00007FF7E5CE83F0]\n\t(No symbol) [0x00007FF7E5B8580A]\n\t(No symbol) [0x00007FF7E5B82F5F]\n\t(No symbol) [0x00007FF7E5B73E49]\n\t(No symbol) [0x00007FF7E5B75BE6]\n\t(No symbol) [0x00007FF7E5B7410F]\n\t(No symbol) [0x00007FF7E5B739BD]\n\t(No symbol) [0x00007FF7E5B738DA]\n\t(No symbol) [0x00007FF7E5B7151E]\n\t(No symbol) [0x00007FF7E5B71DEC]\n\t(No symbol) [0x00007FF7E5B888AA]\n\t(No symbol) [0x00007FF7E5C1C6DE]\n\t(No symbol) [0x00007FF7E5BFBA3A]\n\t(No symbol) [0x00007FF7E5C1B8B3]\n\t(No symbol) [0x00007FF7E5BFB7E3]\n\t(No symbol) [0x00007FF7E5BC75C8]\n\t(No symbol) [0x00007FF7E5BC8731]\n\tGetHandleVerifier [0x00007FF7E607646D+3118813]\n\tGetHandleVerifier [0x00007FF7E60C6CC0+3448624]\n\tGetHandleVerifier [0x00007FF7E60BCF3D+3408301]\n\tGetHandleVerifier [0x00007FF7E5E4A44B+841403]\n\t(No symbol) [0x00007FF7E5CF344F]\n\t(No symbol) [0x00007FF7E5CEF4C4]\n\t(No symbol) [0x00007FF7E5CEF65D]\n\t(No symbol) [0x00007FF7E5CDEBB9]\n\tBaseThreadInitThunk [0x00007FFC817C7374+20]\n\tRtlUserThreadStart [0x00007FFC8281CC91+33]\n\n", "trace": "  File \"C:\\Users\\joshi\\PycharmProjects\\Personal_Projects\\Banking_Project\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\joshi\\PycharmProjects\\Personal_Projects\\Banking_Project\\venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\joshi\\PycharmProjects\\Personal_Projects\\Banking_Project\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\joshi\\PycharmProjects\\Personal_Projects\\Banking_Project\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1139, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\joshi\\PycharmProjects\\Personal_Projects\\Banking_Project\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 890, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\joshi\\PycharmProjects\\Personal_Projects\\Banking_Project\\tests\\tests_Case\\Banking_Project\\pom\\conftest.py\", line 18, in setup\n    driver.get(base_url)\n  File \"C:\\Users\\joshi\\PycharmProjects\\Personal_Projects\\Banking_Project\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 363, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"C:\\Users\\joshi\\PycharmProjects\\Personal_Projects\\Banking_Project\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\joshi\\PycharmProjects\\Personal_Projects\\Banking_Project\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1730460982625, "stop": 1730460988046}], "start": 1730460982625, "stop": 1730460989569}